stations.csv

```diff
@@ -4991,86 +4991,85 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
-6442;Pola de Lena;pola-de-lena;7115122;;;;;f;ES;f;Europe/Madrid;t;ESABM;;f;;f;2001166;f;;f;;f;;f;;f;;f;15122;t;f;;;;Pola De Lena;;;;;;;;;;;;;;
-6443;Palencia;palencia;7114100;;;;;f;ES;f;Europe/Madrid;t;ESABN;;f;;f;7100012;f;;f;;f;;f;;f;;f;14100;t;f;;;;Palencia;;;;;;;;;;;;;;
-6444;Sahagún;sahagun;7115009;;;;;f;ES;f;Europe/Madrid;t;ESABO;;f;;f;7100381;f;;f;;f;;f;;f;;f;15009;t;f;;;;Sahagun;;;;;;;;;;;;;;
+6440;Villanueva del Rio y Minas;villanueva-del-rio-y-minas;7140119;;37.661125;-5.711467;;f;ES;f;Europe/Madrid;t;ESABK;;f;;f;7140119;f;;f;;f;;f;;f;;f;40119;t;f;;;;;;;;;;;;;;;;;;
+6441;Alcolea del Rio;alcolea-del-rio;7140121;;37.6355;-5.73091;;f;ES;f;Europe/Madrid;t;ESABL;;f;;f;;f;;f;;f;;f;;f;;f;40121;t;f;;;;;;;;;;;;;;;;;;
+6442;Pola de Lena;pola-de-lena;7115122;;57.928241;31.982684;;f;ES;f;Europe/Madrid;t;ESABM;;f;;f;2001166;f;;f;;f;;f;;f;;f;15122;t;f;;;;;;;;;;;;;;;;;;
+6443;Palencia;palencia;7114100;;42.01585;-4.533826;;f;ES;f;Europe/Madrid;t;ESABN;;f;;f;7100012;f;;f;;f;;f;;f;;f;14100;t;f;;;;;;;;;;;;;;;;;;
+6444;Sahagún;sahagun;7115009;;42.36954;-5.022291;;f;ES;f;Europe/Madrid;t;ESABO;;f;;f;7100381;f;;f;;f;;f;;f;;f;15009;t;f;;;;;;;;;;;;;;;;;;
 6445;Riaza;riaza;7117109;;;;;f;ES;f;Europe/Madrid;f;ESABP;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
 6446;Lerma;lerma;7117205;;;;;f;ES;f;Europe/Madrid;f;ESABQ;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6447;Villalba de Guadarrama;villalba-de-guadarrama;7110200;;;;;f;ES;f;Europe/Madrid;t;ESABR;;f;;f;7100384;f;;f;;f;;f;;f;;f;10200;t;f;;;;Villalba De Guadarrama;;;;;;;;;;;;;;
-6448;Reinosa;reinosa;7114202;;;;;f;ES;f;Europe/Madrid;t;ESABS;;f;;f;7100163;f;;f;;f;;f;;f;;f;14202;t;f;;;;Reinosa;;;;;;;;;;;;;;
-6449;Ponferrada;ponferrada;7120200;;41.2312;-0.155953;;f;ES;f;Europe/Madrid;t;ESABT;;f;;f;7100190;f;;f;;f;;f;;f;;f;20200;t;f;;;;Ponferrada;;;;;;;;;;;;;;
-6450;A Rúa-Petín;a-rua-petin;7120213;;;;;f;ES;f;Europe/Madrid;t;ESABU;;f;;f;7100387;f;;f;;f;;f;;f;;f;20213;t;f;;;;A Rua-Petin;;;;;;;;;;;;;;
-6451;Monforte de Lemos;monforte-de-lemos;7120300;;41.234616;0.143647;;f;ES;f;Europe/Madrid;t;ESABV;;f;;f;7100020;f;;f;;f;;f;;f;;f;20300;t;f;;;;Monforte De Lemos;;;;;;;;;;;;;;
-6452;Navalperal;navalperal;7110208;;;;;f;ES;f;Europe/Madrid;t;ESABW;;f;;f;7178781;f;;f;;f;;f;;f;;f;10208;t;f;;;;Navalperal;;;;;;;;;;;;;;
-6453;Las Fraguas;las-fraguas;7114209;;;;;f;ES;f;Europe/Madrid;t;ESABX;;f;;f;;f;;f;;f;;f;;f;;f;14209;t;f;;;;Las Fraguas;;;;;;;;;;;;;;
-6454;Alegría de Álava;alegria-de-alava;7111210;;;;;f;ES;f;Europe/Madrid;t;ESABY;;f;;f;7111210;f;;f;;f;;f;;f;;f;11210;t;f;;;;Alegria De Alava-Dulantzi;;;;;;;;;;;;;;
-6455;Oviedo;oviedo;7115211;;;;;f;ES;f;Europe/Madrid;t;ESABZ;;f;;f;7100152;f;;f;;f;;f;;f;;f;15211;t;f;;;;Oviedo;;;;;;;;;;;;;;
-6456;Salvatierra de Álava;salvatierra-de-alava;7111212;;;;;f;ES;f;Europe/Madrid;t;ESACA;;f;;f;;f;;f;;f;;f;;f;;f;11212;t;f;;;;Salvatierra/Agurain;;;;;;;;;;;;;;
-6457;Araya;araya;7111213;;;;;f;ES;f;Europe/Madrid;t;ESACB;;f;;f;;f;;f;;f;;f;;f;;f;11213;t;f;;;;Araia;;;;;;;;;;;;;;
```

Comment:
good question, probably because I was tired :tired_face: 

I’ll add it back with the next batch